### DISCUSSION TITLE
trying to access yougov but doesnt allow through firefox, even with mask on

### DISCUSSION THREAD

**Discussion by Yellowbram** [denschub/chrome-mask#25] [Q&A]
the website is 

https://account.yougov.com/gb-en/account/safe

Should i be able to access using the mask? thanks!

**Comment by skyschub** [comment]
I don't know. Please report your issue with that site on webcompat.com, or via the "Report broken site" feature location in Firefox' hamburger menu. My team will investigate what's happening there.
